### ISSUE TITLE
I cant find SQL CREATE TABLE anywhere

### ISSUE THREAD

**Issue by halojoy** [Phamiliarize/karuiboard#2]
You have the database CREATE TABLE .sql?
Without this your script will be impossible to run.
There are several tables, I understand from the query.

**Comment by Phamiliarize** [comment]
That's correct. My apologies as I know you're excited to add to this project, but at the moment this script isn't even near complete and the repository is really just being used to track my changes and store my code- that way if I break anything I can undo it. (And I'm too cheap to pay for a private repository yet.)

I've been really sick recently and haven't made much progress- today I am also sick and going to be resting all day... (waiting for the clinic atm) but hopefully soon I can get the core functionality of this finished and then I will release the DB along with that (and begin to work on an Install Script.)

I appreciate your interest in contributing with PHP PDO and to the project! I hope that when I have something more complete that interest is still there 😁 🤝

**Comment by halojoy** [comment]
Thar is understood.
Yes, the project is not very complete.
But is a very promising beginning.
I stand by for more code when you feel better and want to continue.

**Comment by Phamiliarize** [comment]
I dunno if it's best to tackle PDO stuff yet, since I don't have all the Data Transactions in existence yet... (Maybe that makes it better? xD) But the SQL dump is now up. (Just a phpmyadmine xport of the structure.)

**Comment by Phamiliarize** [comment]
Hey Halojoy! Thank you for your understanding and your patience.

The code is getting a bit more complex and crazy, but the core functionality is almost all there. (Just have to add the session/logging in and then replace the dummy value in all the CRUD queries.)

That said, i've added a snap shot of the SQL structure.  Are you still interested in collaborating in converting things to PDO?

The next phase I plan is to clean up code pass, and then work on some security things. PDO would probably be good either before or after this.
